AppKit/CPBox.j

```diff
@@ -90,7 +90,8 @@ CPBelowBottom = 6;
             @"inner-shadow-size": 6.0,
             @"inner-shadow-color": [CPNull null],
             @"content-margin": CGSizeMakeZero(),
-        };
+            @"nib2cib-adjustment-primary-frame": CGRectMake(4, -4, -8, -6)
```

Comment:
Why this is not called `nib2cib-adjustment-frame` as it is for other controls ?

Reply:
Well, in fact I wanted to show that this adjustment will only be applied on primary (and secondary) boxes (and not on all types of boxes)... This way, other types may use nib2cib-adjustment-frame the usual way…